Board.cpp

```diff
@@ -153,20 +155,19 @@ bool Board::checkBlockFieldColision() {
 }
 
 void Board::generateNewBlock() {
-	if (bomb) {
-		delete bomb;
-		bomb = nullptr;
-	}
-	else if (block)
+	if (block)
 		delete block;
-	if (steps % 20 == 0) {
-		bomb = new Bomb(Point(boardOffset.getX() + WIDTH / 2 - 2, 0));
-		block = bomb;
+
+	auto chance = getRandom(1, 20);

```

Comment:
we have to draw at 5% chance every time 

_"The chances for a bomb to appear are 5% per each new shape that should appear."_